mux_test.go

```diff
@@ -336,8 +336,8 @@ func TestPath(t *testing.T) {
 			shouldMatch:  false,
 		},
 		{
-			title:        "Path route, URL with encoded slash does match",
-			route:        new(Route).Path("/v1/{v1}/v2"),
+			title:        "Path route with useEncodedPath, URL with encoded slash does match",
+			route:        (&Route{useEncodedPath: true}).Path("/v1/{v1}/v2"),
```

Comment:
I'm all for breaking it out.
On Tue, Aug 30, 2016 at 2:22 PM Kush Mansingh <notifications@github.com>
wrote:

> In mux_test.go
> <https://github.com/gorilla/mux/pull/190#discussion_r76857850>:
>
> > @@ -336,8 +336,8 @@ func TestPath(t *testing.T) {
> >  			shouldMatch:  false,
> >  		},
> >  		{
> > -			title:        "Path route, URL with encoded slash does match",
> > -			route:        new(Route).Path("/v1/{v1}/v2"),
> > +			title:        "Path route with useEncodedPath, URL with encoded slash does match",
> > +			route:        (&Route{useEncodedPath: true}).Path("/v1/{v1}/v2"),
>
> I don't really like this. I'm debating whether it's worth breaking out
> into something like TestSkipClean
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/gorilla/mux/pull/190/files/80a7f1ea478c9ef6768a3e1f24e15b57a6d0f563#r76857850>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABIcPDuuP1tz2imALcD9F21ofwNn210ks5qlH0OgaJpZM4Jw235>
> .
>